packages/atomic/src/components/commerce/atomic-commerce-search-box/e2e/atomic-commerce-search-box.e2e.ts

```diff
@@ -0,0 +1,234 @@
+import {test, expect} from './fixture';
+
+test.describe('default', () => {
+  test.beforeEach(async ({page}) => {
+    await page.goto(
+      'http://localhost:4400/iframe.html?id=atomic-commerce-search-box--default&viewMode=story&args=suggestion-timeout:5000'
+    );
+  });
+
+  test('should have an enabled search button', async ({searchBox}) => {
+    await expect(searchBox.submitButton).toBeEnabled();
+  });
+
+  test('should be A11y compliant', async ({searchBox, makeAxeBuilder}) => {
+    await searchBox.hydrated.waitFor();
+    const accessibilityResults = await makeAxeBuilder().analyze();
+    expect(accessibilityResults.violations).toEqual([]);
+  });
+
+  test.describe('after clicking the searchbox input', () => {
+    test.beforeEach(async ({searchBox}) => {
+      await searchBox.searchInput.waitFor({state: 'visible'});
+      await searchBox.searchInput.click();
+    });
+
+    test('should display suggested queries', async ({searchBox}) => {
+      await expect(searchBox.searchSuggestions().first()).toBeVisible();
+    });
+
+    test('should be A11y compliant', async ({searchBox, makeAxeBuilder}) => {
+      await searchBox.hydrated.waitFor();
+      const accessibilityResults = await makeAxeBuilder().analyze();
+      expect(accessibilityResults.violations).toEqual([]);
+    });
+
+    test.describe('after clicking the submit button', () => {
+      test.beforeEach(async ({searchBox}) => {
+        await searchBox
+          .searchSuggestions()
+          .first()
+          .waitFor({state: 'visible', timeout: 10e3});
+        await searchBox.submitButton.click();
+      });
+
+      test('should collapse the suggested queries', async ({searchBox}) => {
+        await expect(searchBox.searchSuggestions().first()).not.toBeVisible();
+      });
+    });
+  });
+});
+
+test.describe('with instant results & query suggestions', () => {
+  test.beforeEach(async ({page}) => {
+    await page.goto(
```

Comment:
Since this type of URL will be used for every test, we could have a utility function, that accept a component name or tag, and arguments for the component props (strongly typed? ... derived from the component HTML type 🤔 ?).

And navigate to the page or simply return the built string.

Reply:
I thought about it!

Let's see that in a future iteration.

Reply:
https://coveord.atlassian.net/browse/KIT-3241